<launch>
    <node pkg="scqn_sam" type="scqn_localization_node" name="scqn_localization_node" output="screen">
        <rosparam file="$(find scqn_sam)/launch/scqn_localization_params.yaml" command="load" />
    </node>
</launch>
